gix-diff/tests/diff/blob/slider.rs

```diff
@@ -80,13 +81,39 @@ fn baseline() -> gix_testtools::Result {
             })
             .to_string();
 
-        pretty_assertions::assert_eq!(actual, baseline);
+        let actual_matches_baseline = actual == baseline;
+        diffs.push((actual, baseline, actual_matches_baseline, file_name));
     }
 
-    if count == 0 {
+    if diffs.is_empty() {
         eprintln!("Slider baseline isn't setup - look at ./gix-diff/tests/README.md for instructions");
     }
 
+    let total_diffs = diffs.len();
+    let matching_diffs = diffs
+        .iter()
+        .filter(|(_, _, actual_matches_baseline, _)| *actual_matches_baseline)
+        .count();
+
+    assert!(
+        matching_diffs == total_diffs,
+        "assertion failed: total diffs {} == matching diffs {}\n\n{}",
```

Comment:
I think the assertion failure will come across by merit of `assert_eq`, so the text can skip `assertion failed:`.